connectors/standalone/src/main/scala/io/delta/standalone/internal/OptimisticTransactionImpl.scala

```diff
@@ -411,7 +411,7 @@ private[internal] class OptimisticTransactionImpl(
    * next.
    *
    * @throws DeltaConcurrentModificationException if the next commit version cannot be resolved
-   *                                              due to the concurrent modifications in the
+   *                                              due to the concurrent modification in the
```

Comment:
In my previous nit I was referring to `in` not the plural of `modification` 😉  I'm OK with `modification` (singular).

nit: s/modification in/modification to

Reply:
Sorry, I didn't get the nit syntax right :pray: Should be fixed now in https://github.com/delta-io/delta/pull/1947/commits/43c617d28f5559dfa99a99f914d4ed07f38e1d4e